input_files/ska_test_file.txt

```diff
@@ -13,10 +13,16 @@ periodicity_sigma_cutoff 20
 periodicity_harmonics 32
 analysis
 -acceleration
+-acceleration_jerk
```

Comment:
Maybe we already talked about it. Just a remainder to be sure. This changes needs to be included in the header file used in autotune script

Reply:
Done with KA merge